specification/storage/resource-manager/Microsoft.Storage/stable/2021-06-01/blob.json

```diff
@@ -325,6 +325,9 @@
         "x-ms-examples": {
           "GetContainers": {
             "$ref": "./examples/BlobContainersGet.json"
+          },
```

Comment:
WE should also add "AllowProtectedAppendWritesAll" to account level worm.

Reply:
Not supported right now